src/components/dashboard/FaceRecognition/FaceRecognition.js

```diff
@@ -176,12 +138,12 @@ class FaceRecognition extends React.Component<FaceRecognitionProps, State> {
           />
         )}
 
-        {this.state.zoomReady && showZoomCapture && (
-          <ZoomCapture
-            screenProps={this.props.screenProps}
-            onCaptureResult={this.onCaptureResult}
-            showZoomCapture={this.state.zoomReady && showZoomCapture}
-            loadedZoom={this.loadedZoom}
+        {showCamera && (
+          <FaceCapture
+            pictureOptions={{
```

Comment:
remove pictureOptions prop as we've removed it from the FaceCapture component contract.
it uses 1080 for final shoot and 480 for shoots made to validate for face position by default

you can specify new props values to demonstrate a new component's contract:

```
<FaceCapture
  photos={2} // count of the final face photos to make
  quality={1080} // quality of the final photos
  sampleQuality={480} // quality of the transitional photos made for validate face position
  onFaces=// ...
```

Reply:
but all this is passed on to default